python/lsst/atmospec/spectraction.py

```diff
@@ -405,44 +414,10 @@ def run(self, exp, xpos, ypos, target, outputRoot=None, plotting=True):
         # Full forward model extraction:
         # adds transverse ADR and order 2 subtraction
         w = None
-        if parameters.PSF_EXTRACTION_MODE == "PSF_2D" and parameters.OBS_OBJECT_TYPE == "STAR":
-            w = FullForwardModelFitWorkspace(spectrum, verbose=1, plot=True, live_fit=False,
+        if parameters.SPECTRACTOR_DECONVOLUTION_FFM:
```

Comment:
Should this be mutually exclusive with the `parameters.SPECTRACTOR_DECONVOLUTION_PSF2D`?  As is, it looks like you could potentially attempt to deconvolve this twice.

Reply:
Agreed. I can raise that with Jérémy, but this top level file is designed to just follow the top-level workflow from upstream, and I've checked and that's how things are there. Will see if he just treats them as mutually exclusive though, or if they are ever run together or what.